src/tests/BaseCard.test.js

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from 'redux/store';
+import BaseCurrencyContext from 'context/BaseCurrencyContext';
+import { BrowserRouter as Router } from 'react-router-dom';
+import BaseCard from 'components/BaseCard';
+
+describe('Tests for <BaseCard /> component', () => {
+  const base = (
+    <BaseCard
+      baseCurrency="USD"
+    />
+  );
+  const baseCurrency = 'USD';
+
+  const wrap = (component) => (
+    <Provider store={store}>
+      <BaseCurrencyContext.Provider value={{ baseCurrency }}>
+        <Router>
+          {component}
+        </Router>
+      </BaseCurrencyContext.Provider>
+    </Provider>
+  );
+  it('Should render the content correctly', () => {
+    render(wrap(base));
+    expect(screen.getByText('USD')).not.toBeNull();
+  });
+
+  it('Should match the snapshot', () => {
+    expect(renderer.create(wrap(base)).toJSON()).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Good job setting up the testes 👍 , however all the testes you set up are not passing. Kindly ensure that all your testes are passing as you know testes are very important when you are developing a product, they will help you to debug in the future 😃 

![Capture](https://user-images.githubusercontent.com/80458199/225969197-e7eda893-3103-4631-b9b0-06fc016bed85.PNG)

Reply:
I checked all the tests. For a silly mistake, one failed. I fixed it. Thanks.